readme.md

```diff
@@ -43,6 +43,12 @@ Type: `Object`
 Type: `boolean`<br>
 Default: `true`
 
+##### mtime
+
+If this optional field is specified, then its value will be assigned as the modification timestamp for all files added to the zip archive. Setting it to an invariant across executions will make the resulting zip archive's hash a means for detecting file content changes.
```

Comment:
I think this sentence could be written more succinctly:

> Setting it to an invariant across executions will make the resulting zip archive's hash a means for detecting file content changes.

Don't use words like invariant.

Reply:
How about this? It's actually _less_ succinct, but it avoids the word "invariant" and is more direct about what is happening and why it could be beneficial. 

> Setting it to the same value across executions is a way to create stable zip archives--archives that change only when the contents of their entries change, regardless of whether those entries were "touched" or regenerated.